### DISCUSSION TITLE
Allow import modules from VSCode

### DISCUSSION THREAD

**Discussion by cvelasco88** [microsoft/vscode-jupyter#9345] [New Ideas]
As it's shown in the video of this guy, the default projects which doesn't have the libraries downloaded can't run properly until is installed the lib on the enviroment. It would be great have a quick option (kind it does IntelliJ).
The error of library missing it helps poorly, so it could be better.

https://www.youtube.com/watch?v=ThU13tikHQw&t=177s

**Comment by DonJayamanne** [comment]
Thanks for the suggestion, closing this as a duplicate of https://github.com/microsoft/vscode-jupyter/issues/1566
